src/excmds.ts

```diff
@@ -370,6 +373,103 @@ export async function reader() {
     }
 }
 
+/** The kinds of input elements that we want to be included in the "focusinput"
+ * command (gi)
+ */
+export const INPUTTAGS_selectors = `
+input:not([type]),
+input[type='text'],
+input[type='search'],
+input[type='password'],
+input[type='file'],
+input[type='datetime'],
+input[type='datetime-local'],
+input[type='date'],
+input[type='month'],
+input[type='time'],
+input[type='week'],
+input[type='number'],
+input[type='range'],
+input[type='email'],
+input[type='url'],
+input[type='tel'],
+input[type='color'],
+textarea:not([disabled]):not([readonly])
```

Comment:
Consider using the `:-moz-any()` pseudo selector (the vendor prefixed version of `:match()`) to make this shorter and to disallow disabled and readonly inputs.

```
input:not([disabled]):not([readonly]):-moz-any(:not([type]), [type='text'], [type='search'], ...)
```

the `isTextEditable` function in dom.ts also identifies `object` nodes and `[role="application"]` as editable elements, though I've never noticed an object node in the wild and there's nothing about their spec that requires them to be editable. 


document.querySelectorAll('h4:-moz-any(.browser--safari, .browser--firefox)')